util/stmtsummary/statement_summary.go

```diff
@@ -721,6 +726,16 @@ func (ssElement *stmtSummaryByDigestElement) add(sei *StmtExecInfo, intervalSeco
 		commitDetails.Mu.Unlock()
 	}
 
+	//plan cache
+	if sei.StmtCtx.IsExecute { //For the execution of prepared statements
```

Comment:
We don't need to add `IsExecute` flag actually, because `FoundInPlanCache` can only be true when it is `execute`?